### PR TITLE
#28 - landing page

### LINKED ISSUE #28: Create the landing page
As in title. We want user to see the landing page with some background and possibly buttons for user to choose his faculty/department

### PR DESCRIPTION
**Pull request by brzozka**
This PR should close #28 

Color scheme/pictures are not final, so they can be changed later on. For now  buttons redirect user to empty home page, cause there is no other page yet. Funcionality will be implemented later into the project.

### CODE REVIEW

**Comment by krzysztofrewak** [comment]
Btw, the good practice would be adding some screenshots of your work.

**Comment by brzozka** [comment]
I forgot to add the screenshot of the page, and my laptop started lagging, that's why it took so long, but here it is.
image

**Comment by krzysztofrewak** [comment]
Maybe something like this would be better? Or at least more consistent?

image

File: frontend/main.js [hunk 1]
```diff
@@ -2,5 +2,6 @@ import { createApp } from "vue";
 import App from "./App.vue";
 import router from "./router";
 import store from "./store";
+import "./assets/tailwind.css";
```
Comment (mtracz) [inline frontend/main.js:5]: Please use `@` alias instead of `./`

**Review by mtracz — CHANGES_REQUESTED** [review]

File: frontend/main.js (hunk 1 above)
Comment (brzozka) [reply frontend/main.js:5]: Oh yeah, I forgot about it, thanks

**Comment by brzozka** [comment]
Something like this?
image

**Review by mtracz — APPROVED** [review]

**Review by krzysztofrewak — APPROVED** [review]

**Review by EwelinaLasowy — APPROVED** [review]

**Pull request merged**